src/Symfony/Bundle/FrameworkBundle/Command/AssetsInstallCommand.php

```diff
@@ -33,10 +34,40 @@ class AssetsInstallCommand extends ContainerAwareCommand
     const METHOD_ABSOLUTE_SYMLINK = 'absolute symlink';
     const METHOD_RELATIVE_SYMLINK = 'relative symlink';
 
+    private $baseDir;
+    private $filesystem;
+
     /**
-     * @var Filesystem
+     * @param string          $baseDir
+     * @param Filesystem|null $filesystem
      */
-    private $filesystem;
+    public function __construct($baseDir = null, Filesystem $filesystem = null /**, $newApi = false*/)
```

Comment:
letting a dead arg being passed is weird, adding a real optional argument once this one removed would break the subclass. I would definitely make $filesystem the first arg again and don't break anything instead.